specs/unified_kernel_image.md

```diff
@@ -159,3 +159,22 @@ that tools staging or consuming UKIs have a common place to store and look for t
 
 The installed UKIs should have a filename `<version format specification>.efi`, i.e. the filename is left to
 implementers but must be valid for comparisons according to the [Version Format Specification](version_format_specification.md).
+
+UKI addons (cmdline PE addons, systemd-sysext extensions for the initrd) built centrally by distributions and
```

Comment:
hmm, please don't call systemd-sysext stuff "addons".

addons → those efi PE wrapped things
sysexts → dm-verity enabled GPT disk images (DDIs)

also, we should cover confext here too, it's a thing now. And a pretty useful one at that.

Hence maybe say:

> Auxiliary UKI resources (such as PE addons for command line extensions and similar, as well as systemd-sysext and systemd-confext DDIs) …